plugins/woocommerce/src/Blocks/Patterns/PatternRegistry.php

```diff
@@ -169,7 +169,7 @@ function ( $property ) {
 				register_block_pattern_category(
 					$category_slug,
                     // phpcs:ignore WordPress.WP.I18n.NonSingularStringLiteralText
-					array( 'label' => __( $category, 'woocommerce' ) )
+					array( 'label' => __( self::kebab_to_capital_case( $category_slug ), 'woocommerce' ) )
```

Comment:
I think that we can avoid to invoke `__`:WDYT?

Reply:
Aslo, we shouldn't put variable directly inside i18n functions. See https://developer.wordpress.org/apis/internationalization/internationalization-guidelines/#variables

Reply:
Yeah!